spa/src/app/files/table/table.component.ts

```diff
@@ -32,7 +32,7 @@ export class TableComponent implements OnInit {
     displayedColumns = [
         "program", "project", "submittedDonorId",
         "submittedSpecimenId", "specimen_type", "submittedSampleId",
-        "software", "title", "dos_uri", "fileSize"
+        "aliases", "title", "dos_uri", "fileSize"
```

Comment:
I order important here, and if so, is this the order we want? It looks a bit different than the order we are using elsewhere.